### PR TITLE
Added Nagarro into this 

### PR DESCRIPTION
**Pull request by shivam-jha2712**
📝Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]`

ℹ️Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

- [x] This PR contains housekeeping only (URL edits, copy changes etc) - if check, delete other lines that don't apply.
- [ ] I am an employee of the company mentioned and confirm all included details are correct.
- [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [ ] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md) is included - This is __required__ for new additions.
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/main/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by dougaitken** [comment]
@shivam-jha2712 Can you review the company profile, please? There are some odd line breaks and grammar to check. Thanks.

**Comment by shivam-jha2712** [comment]
Sure will do the same

**Comment by shivam-jha2712** [comment]
> @shivam-jha2712 Can you review the company profile, please? There are some odd line breaks and grammar to check. Thanks.

If you could review it and merge it please

File: company-profiles/nagarro.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Nagarro 
+
+## Company blurb
+
+[Nagarro](https://www.nagarro.com/en) | We excel at digital product engineering. We solve complex business challenges through agility and innovation. We call it 
+
+thinking breakthroughs. see how we have helped industry leaders embrace digitalization and accelerate technology-led innovation. We care. Caring is our superpower.
```
Comment (dougaitken) [inline company-profiles/nagarro.md:7]: There is an odd line break between `We call it` and `thinking`

File: company-profiles/nagarro.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+itdrives us to deliver excellence to our clients.
```
Comment (dougaitken) [inline company-profiles/nagarro.md:9]: `It drives` ?

File: company-profiles/nagarro.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Remote status
+
+We believe in freedom! With this, Nagarrian can work from wherever they want, whether it's a home office, in our offices or a hybrid model.
+
+[Careers](https://www.nagarro.com/en/careers)
+
+## Region
+
+Brazil.
```
Comment (dougaitken) [inline company-profiles/nagarro.md:25]: Readme says Worldwide. This section is for where in the world the company hires, is it only Brazil, or Worldwide?

File: company-profiles/nagarro.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- Swift
+- low code
+- Blockchain 
+- Cloud
+
+## Office locations
+
+Our offices are located in
+
+Austria, Mexico, Sri Lanka, Germany, Poland, India, USA, Romania
```
Comment (dougaitken) [inline company-profiles/nagarro.md:42]: Another odd line break.

File: company-profiles/nagarro.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+## How to apply
+
+We are higin, visit [Nagarro](https://www.nagarro.com/en/careers) or [Linkedin](https://www.linkedin.com/company/nagarro/)
```
Comment (dougaitken) [inline company-profiles/nagarro.md:46]: `hiring` ?

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Comment by shivam-jha2712** [comment]
Hey! Have you reviewed it ?

**Comment by shivam-jha2712** [comment]
I have completed the said changes could you please review this and merge it

**Comment by shivam-jha2712** [comment]
Hey Could you please review this and help me with my last PR

**Comment by dougaitken** [comment]
If this is for Hacktoberfest @shivam-jha2712 then please be patient, there is no rush.

File: company-profiles/nagarro.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+# Nagarro 
+
+## Company blurb
+
+[Nagarro](https://www.nagarro.com/en) | We excel at digital product engineering. We solve complex business challenges through agility and innovation. We call it 
+thinking breakthroughs. see how we have helped industry leaders embrace digitalization and accelerate technology-led innovation. We care. Caring is our superpower.
```
Comment (dougaitken) [inline company-profiles/nagarro.md:6]: Unneeded line break.

File: company-profiles/nagarro.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+- Go
+- Swift
+- low code
+- Blockchain 
+- Cloud
+
+## Office locations
+
+Our offices are located in
+Austria, Mexico, Sri Lanka, Germany, Poland, India, USA, Romania
```
Comment (dougaitken) [inline company-profiles/nagarro.md:39]: Unneeded line break

**Review by dougaitken — CHANGES_REQUESTED** [review]

**Review by shivam-jha2712 — COMMENTED** [review]
Have made the changes removed all the spaces

**Review by dougaitken — APPROVED** [review]

**Pull request merged**